services/core-web/common/reducers/staticContentReducer.js

```diff
@@ -37,6 +37,7 @@ const initialState = {
   bondStatusOptions: [],
   bondTypeOptions: [],
   bondDocumentTypeOptions: [],
+  exemptionFeeSatusOptions: [],
```

Comment:
Satus --> Status :D
(In a few locations)

Reply:
Did I really do that and then copy and paste that EVERYWHERE? Yes. Fixed!